packages/graphql/src/schema/to-compose.ts

```diff
@@ -86,7 +86,7 @@ export function objectFieldsToComposeFields(fields: BaseField[]): {
 
 export function objectFieldsToCreateInputFields(fields: BaseField[]): Record<string, InputField> {
     return fields
-        .filter((f) => !(f as PrimitiveField)?.autogenerate && !(f as TemporalField)?.timestamps)
+        .filter((f) => !(f as PrimitiveField)?.autogenerate && !(f as PrimitiveField)?.callback && !(f as TemporalField)?.timestamps)
```

Comment:
Maybe it is worth using intermediate variables to split the complexity of this boolean logic?

Something akin to:
```suggestion
        .filter((f) => {
        const isAutogenerate=(f as PrimitiveField)?.autogenerate
        const isCallback=(f as PrimitiveField)?.autogenerate
        const isTemporal=(f as TemporalField)?.timestamps
       
        return !isAutogenerate && !isCallback && !isTemporal
        });
```

It is more verbose, but I think it helps a lot in readability